Makefile

```diff
@@ -10,15 +10,11 @@
 
 MKDOCS_IMAGE ?= asc-mkdocs
 
-build-all: build-soc build-dgpu
```

Comment:
so build-all should build both dlstreamer without realsense and with realsense? what's the definition of build-all now

Reply:
Would it make sense to also add the OVMS docker builds and use this as a true build all